packages/fx-core/src/plugins/resource/sql/utils/message.ts

```diff
@@ -28,24 +24,4 @@ export class Message {
 
   public static readonly adminName = (name: string) =>
```

Comment:
do we need to localize `checkAadAdmin`, `connectDatabase`, `adminName` messages?